lib/messages/user-settings/index.js

```diff
@@ -36,7 +36,7 @@ class CombinedSettings extends Message {
       });
     }
 
-    if (this.muteUnfurlPromptsUntil || this.muteUnfurlPromptsIndefinitely) {
+    if ((this.muteUnfurlPromptsUntil > moment.unix()) || this.muteUnfurlPromptsIndefinitely) {
```

Comment:
Should this comparison sign be the other way around based on our discussion here?

https://github.com/integrations/slack/issues/575#issuecomment-419236694

Reply:
Ah, no. I think I'm wrong 🤔

Reply:
> _Should this comparison sign be the other way around based on our discussion here?_

I thought the same thing initially! But then I realized that entire conversation was us deciding on what we should do to **not** show the setting. Sooo for the implementation, I had to _negate_ our discussion logic (it confused me for quite a bit 😆 )

> Ah, no. I think I'm wrong

You're not wrong, per se, it's just confusing perspective!